workflow-templates/im-run-flyway-repair.yml

```diff
@@ -129,8 +130,11 @@ jobs:
       #       database/static-creds/<role-name> username | SQL_USERNAME ;
       #       database/static-creds/<role-name> password | SQL_PASSWORD
 
+      # TODO: If your database uses Azure Active Directory Authentication then you can login with a Service Principal
+      #       or Managed Service Identity instead of using the username and password of an account stored in Key Vault.
```

Comment:
Logging in with a managed service identity wouldn't be appropriate from our GitHub action runners. Only a service principal would make sense.